software/cloud-seeder.xml

```diff
@@ -0,0 +1,11 @@
+name: Cloud Seeder
+website_url: https://ipv6.rs/cloudseeder/
+description: Cloud Seeder is a 1-click server appliance deployment and maintenance suite for Windows, MacOS and Linux.
+licenses:
+  - ⊘ Proprietary
+platforms:
+  - Go
+  - Podman
```

Comment:
```suggestion
```
Does this software work with Docker? if so, you can add docker here. Otherwise, podman isn't added yet. that's why it errors out.

Reply:
Hi, as discussed in https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/856, I think it's better/easier just to remove `Podman` for the time being.